### PR TITLE
Go to source file of the component

### PR DESCRIPTION
**Pull request by sumitarora**
Fixes: https://github.com/rangle/batarangle/issues/34

### CODE REVIEW

File: src/frontend/components/component-info/component-info.ts [hunk 1]
```diff
@@ -32,6 +32,23 @@ export default class ComponentInfo {
...
+  viewComponentSource($event) {
+    const highlightStr = '[batarangle-id=\"' + this.node.id + '\"]';
+
+    let evalStr = `inspect(ng.probe(document.querySelector('${highlightStr}'))
+    .componentInstance.constructor)`;
+
+    chrome.devtools.inspectedWindow.eval(
+      evalStr,
+      function(result, isException) {
+        console.log(result, isException);
```
Comment (vanessayuenn) [inline src/frontend/components/component-info/component-info.ts]: `result` and `isException` are mutually exclusive though

Comment (sumitarora) [inline src/frontend/components/component-info/component-info.ts]: yes they are

Comment (vanessayuenn) [inline src/frontend/components/component-info/component-info.ts]: so why are we logging them both at the same time?

Comment (sumitarora) [inline src/frontend/components/component-info/component-info.ts]: This opens source panel for the file we don't want to do anything just display response error or result

Comment (vanessayuenn) [inline src/frontend/components/component-info/component-info.ts]: i think we can just log the errors if there is any.

Comment (sumitarora) [inline src/frontend/components/component-info/component-info.ts]: done

**Comment by vanessayuenn** [comment]
LGTM :+1:

**Pull request merged**